Admin/StaticPageAdmin.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Grossum\StaticPageBundle\Admin;
+
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
+
+use Sonata\AdminBundle\Admin\Admin;
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+
+use Ivory\CKEditorBundle\Form\Type\CKEditorType;
+
+class StaticPageAdmin extends Admin
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        $formMapper
+            ->add('enabled')
```

Comment:
Please, add title to field. Ithink it should be for all fields.
Please try to add title from translation file.